compiler/src/yul/mappers/expressions.rs

```diff
@@ -62,6 +62,27 @@ pub fn expr(
     }
 }
 
+
+pub fn expr_bin_operation<'a>(exp: &Spanned<vyp::Expr<'a>>) -> Result<ExtExpression, CompileError> {
+    if let vyp::Expr::BinOperation{ left, op, right } = &exp.node {
+        // FIXME: How do I get what I need for the `add`
+        let yul_left = literal_expression! {(left.node)};
+        let yul_right = literal_expression! {(right.node)};
+        return match op.node {
+            vyp::BinOperator::Add => {
+                Ok(ExtExpression {
+                    expression: expression! { add([yul_left], [yul_right]) },
+                    location: Location::Memory,
```

Comment:
For now, U256 values on their own are always stored as values. So we'll need: `Location::Value`.

Reply:
I looked at this a bit more. Any location would work here :| since we're not checking the location when [encoding](https://github.com/ethereum/fe/blob/4bd735adac4df4f7cf805a07de7a2348aee6ec0a/compiler/src/yul/namespace/types.rs#L101) base types, but it should still be `Location::Value`.

Reply:
Yep, I'll change that.